docs/components/input.md

```diff
@@ -33,6 +33,21 @@ If the label is omitted it can be handled with another component, like **KLabel*
 </template>
 ```
 
+### labelAttributes
```

Comment:
If these map 1:1 to `KLabel` props, should it be renamed `labelProps` instead to make it super-transparent?

Same comment applies to all instances in the PR.

Reply:
You could also do something like apply a `data-testid` to the label. This also follows the pattern we use in `KSelect` for `kpopAttributes` prop.